### ISSUE TITLE
Doesn't load config file if invoked from another directory

### ISSUE THREAD

**Issue by Stephen304** [Stephen304/bitcannon#27]
Only affects users who invoke BitCannon from the command line. Not high priority.

**Comment by Yamakaky** [comment]
You should add a config switch, like `-c/--config <file>`, with a default to `./config.json` then `/etc/bitcannon.json`.

**Comment by Yamakaky** [comment]
Sorry, I didn't understand the bug you reported. Still, my comment is valid.

**Comment by Yamakaky** [comment]
BTW, this bug is present if you use a daemon manager like sysvinit or systemd. Cf my proposition.

**Comment by Stephen304** [comment]
Yes, I figure it's the biggest problem for linux users who may invoke it from command line. I'd like to switch to using codegangsta/cli for command line flags / oprions before i start adding more of them to keep it from getting messy. using cli has been on my tasklist for a while so I may bump it up in priority.

**Comment by Stephen304** [comment]
Also I think the real solution is to figure out how to reliably read from files in the same directory as the binary. 

There are a couple solutions but one guy claims that the other solutions are not consistently cross platform. I'll need to test this out sometime.

http://stackoverflow.com/questions/18537257/golang-how-to-get-the-directory-of-the-currently-running-file
